File: integration-test/test-projects/jetifier/app/src/main/java/com/example/app/MainActivity.java
package com.example.app;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.widget.TextView;
import com.example.androidlib.Greetings;
import com.example.generated.GeneratedRegistry;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        TextView helloText = (TextView) findViewById(R.id.helloText);
        helloText.setText(new Greetings().getGreetings(getApplicationContext()));

        TextView helloTextWithAnnotationProcessor =
                (TextView) findViewById(R.id.helloTextWithAnnotationProcessor);
        helloTextWithAnnotationProcessor.setText(new GeneratedRegistry().getGreetings());
    }
}

File: integration-test/test-projects/databindingAndKotlin/app/src/main/kotlin/com/example/android/kotlin/MainActivity.kt
package com.example.android.kotlin

import android.app.Activity
import android.os.Bundle
import android.view.View
import android.view.LayoutInflater
import android.widget.Button
import android.widget.TextView
import com.example.android.kotlin.databinding.ActivityLayoutBinding

class MainActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val binding : ActivityLayoutBinding = ActivityLayoutBinding
                .inflate(LayoutInflater.from(this))!!
        setContentView(binding.root)
        binding.model = ViewModel("foo", "bar")
    }
}
